File: src/main/java/org/example/kotlinWorkshop/java/lesson03/t3_set/ClassWithSets.java
package org.example.kotlinWorkshop.java.lesson03.t3_set;

import java.util.HashSet;
import java.util.Set;

public class ClassWithSets {
  private Set<String> immutableSet;
  private Set<String> mutableSet;

  public ClassWithSets() {
    immutableSet = Set.of("hi", "hello");
    mutableSet = new HashSet<>(Set.of("hi", "hello"));


    mutableSet.add("hi");
    immutableSet.add("hi");
  }
}

File: src/main/kotlin/org/example/kotlinWorkshop/kotlin/lesson03/t3_set/ClassWithSets.kt
package org.example.kotlinWorkshop.kotlin.lesson03.t3_set

class ClassWithSets() {
    val immutableSet: Set<String> = setOf("hi", "hello")
    val mutableSet: MutableSet<String> = mutableSetOf("hi", "hello")

    init {
        mutableSet.add("hi");
        //do not exist
        //immutableSet.add("hi");
    }

}